pkg/scheduler/algorithm/predicates/csi_volume_predicate.go

```diff
@@ -61,37 +64,37 @@ func (c *CSIMaxVolumeLimitChecker) attachableLimitPredicate(
 		return true, nil, nil
 	}
 
-	nodeVolumeLimits := nodeInfo.VolumeLimits()
-
-	// if node does not have volume limits this predicate should exit
-	if len(nodeVolumeLimits) == 0 {
-		return true, nil, nil
-	}
-
 	// a map of unique volume name/csi volume handle and volume limit key
 	newVolumes := make(map[string]string)
```

Comment:
I think it's better to check if nodeInfo is nil here as well before passing on the pointer.